src/PostgREST/App.hs

```diff
@@ -206,8 +204,13 @@ app dbStructure conf apiRequest =
         else return notFound
 
     (ActionRead, TargetRoot, Nothing) -> do
-      body <- encode <$> H.query schema accessibleTables
-      return $ responseLBS status200 [jsonH] $ cs body
+      let encodeApi ti = encodeOpenAPI ti host port
```

Comment:
Yes, that will work

> On 18 Jun 2016, at 16:37, dayou <notifications@github.com> wrote:
> 
> In src/PostgREST/App.hs:
> 
> > @@ -206,8 +204,13 @@ app dbStructure conf apiRequest =
> >          else return notFound
> >  
> >      (ActionRead, TargetRoot, Nothing) -> do
> > -      body <- encode <$> H.query schema accessibleTables
> > -      return $ responseLBS status200 [jsonH] $ cs body
> > +      let encodeApi ti = encodeOpenAPI ti host port
> Right, something was blocking my mind on that.
> A header such as X-Forwarded-URI should take precedence over host and port for feeding information to the generated swagger api.
> location /api/v1 {
> proxy_pass http://Host:Port/;
> proxy_set_header X-Forwarded-URI $request_uri;
> }
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>